guide/english/php/sessions/index.md

```diff
@@ -82,3 +82,14 @@ Last but not least it's important to use php sessions securely. Read our article
 
 #### More Information:
 * <a href="https://secure.php.net/manual/en/book.session.php">php.net session manual</a>
+
+# library session in codeigniter
+codeigniter have library session who help us for making session
```

Comment:
this line could be more grammatically correct.
`codeigniter have library session who help us for making session` ->
`codeigniter provides us with more functionality with sessions with it's in-built libraries session functions. `

Reply:
How about a more concise version like:
> codeigniter has a built-in sessions library which makes it easy to work with sessions.`

Reply:
Yes @RandellDawson and how about in the end we could make it like:- 
`easy to work with sessions` -> `easy to work with sessions and also add more functionality`

Reply:
as this not only makes default session handling easier but also allows for some additional functionality, so this sounds a little better to me.

Reply:
Maybe give a few examples in parentheses like:
> easy to work with sessions and also adds more functionality (i.e. list a couple of the big additions here).